app/Services/ForwardChatworkService.php

```diff
@@ -117,4 +130,52 @@ function ($match) use ($params, &$isMatching) {
 
         return '';
     }
+
+    public function sendMessages($messages, $payload_history_id)
+    {
+        $bot = $this->webhook->bot;
+        $room_id = $this->webhook->room_id;
+        $chatwork = Chatwork::withAPIToken($bot->bot_key);
+
+        foreach($messages as $key=>$message) {
+            try {
+                $chatwork->room($room_id)->messages()->create($message);
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Variables.ValidVariableName.NotCamelCaps</br>**Severity**: error</br>**File**: app/Services/ForwardChatworkService.php</br>**Line**: 142</br>Variable "room_id" is not in valid camel caps format